### PR TITLE
[16.0][MIG] delivery_schenker 

### PR DESCRIPTION
**Pull request by ferran-S73**
Migration from 14.0

Added https://github.com/OCA/delivery-carrier/commit/1abdeaa4ff544cfa5b34a78634a9f3ab6274f829

### CODE REVIEW

**Review by hildickethan — APPROVED** [review]

**Comment by hildickethan** [comment]
code+functional

**Comment by ferran-S73** [comment]
@OCA/logistics-maintainers

File: delivery_schenker/migrations/16.0.1.0.0/post-migration.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Copyright 2023 Studio73 - Ferran Mora
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    for i in range(1, 23):
+        xml_id = "delivery_schenker.schenker_packaging_{}".format(str(i).zfill(2))
```
Comment (sebalix) [inline delivery_schenker/migrations/16.0.1.0.0/post-migration.py:10]: Is this migration script working as expected? I would have thought this XML-ID doesn't exist anymore in post migration, unless such references are dropped right after the full module upgrade?

**Review by sebalix — APPROVED** [review]

**Comment by ferran-S73** [comment]
@dreispt could you try to merge please?

**Review by jbaudoux — APPROVED** [review]

**Pull request merged**